mne/export/tests/test_export.py

```diff
@@ -55,11 +55,14 @@ def test_export_raw_eeglab(tmpdir):
                     reason='edflib-python not installed')
 def test_integer_sfreq_edf(tmp_path):
     """Test saving a Raw array with integer sfreq to EDF."""
-    np.random.RandomState(12345)
+    prng = np.random.RandomState(12345)
```

Comment:
```suggestion
    rng = np.random.RandomState(12345)
```
we use rng everywhere in mne